storyblok/components.json

```diff
@@ -1615,6 +1615,49 @@
       "real_name": "image_block",
       "component_group_uuid": null
     },
+    {
+      "name": "image_block_brand_pivot",
+      "display_name": null,
+      "schema": {
+        "image": {
+          "type": "image",
+          "display_name": "",
+          "description": "Around 1200x500 px",
+          "pos": 0,
+          "translatable": true
+        },
+        "caption": {
+          "type": "text",
+          "pos": 1,
+          "translatable": true
+        },
+        "caption_shadow": {
+          "type": "boolean",
+          "pos": 2
+        },
+        "tab-9266977a-879b-4499-9cef-2e7948d2ca8b": {
+          "type": "tab",
+          "display_name": "Styling",
+          "keys": [
+            "extra_styling"
+          ],
+          "pos": 0
+        },
+        "extra_styling": {
```

Comment:
shall we include this in the block, or remove it?

Reply:
Let's remove it 👍